### PR TITLE
resetting the local storage

### PR DESCRIPTION
**Pull request by Have-Samuel**
Awesome Books Project We:
1. Applied Git and Github workflow
2. Used best HTML & CSS practices
3. No linter errors
4. Created a bookList app dynamically

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi Team 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



_**You have submitted a project, that has no linter setup or properly documented**_

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by BenMukebo** [comment]
Any problem Kemi
?

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi ** @Kemi, **

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💪🏻

- Good commit messages ✔️ 
- You have followed  JavaScript best practices. ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


- [ ] Great job on adding linters to your project👏 . However, they are not passing yet, kindly fix this issue it is always better to have all linter checks passing, this helps you avoid errors and enforces conventions.

- [ ] Kindly, Add a descriptive README file to your project as required please use this [template](https://github.com/microverseinc/readme-template) You can copy past but remember to customize it to your project.

- [ ] Please, add the `start` script in your package.json, as for now, I can not run your project
start


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi @Have-Samuel, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._
- Let's make the changes based on what the previous reviewer advised. I haven't seen any change since the last reviewer left a comment.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "PortfolioCsT1",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "script.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (BenMukebo) [inline package.json:7]: - [OPTIONAL ] As I mentioned before, the script `start` is missing in your package.json to run your project, see this image, when I try to run `npm start` for this project, it says this 

start

- This review is optional but I strongly recommend you take this into consideration and make this change.

**Review by BenMukebo — APPROVED** [review]
Hi *** @kemigabocatherine , *** 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**